apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/filter-table-data.ts

```diff
@@ -1,9 +1,24 @@
+import { IMultilingualText } from '@coscrad/api-interfaces';
 import { isNonEmptyString, isNull, isUndefined } from '@coscrad/validation-constraints';
 
 export type Matchers<T> = {
     [K in keyof T]?: (value: T[K], searchTerm: string) => boolean;
 };
 
+// Break out into separate file and test?
+const isMultilingualText = (text: unknown): text is IMultilingualText => {
```

Comment:
Don't do this here. You are breaking the contract with the designer of the index table and also doing a lot of unnecessary run-time type checking. In order to be intentional, we want to configure the multilingual text property search at the level of the index view.